process faults {5: ({6, 2}, 2641883225)}
network faults {}
{7: ("628A18B548A1F306852EF339B310D373150BD138CA936C2A28F74012406203C1", {3, 5, 6, 0, 4, 1, 2}), 5: ("1360EFFA30A2D86CDD45ADE6FDE2A838F89DBD8D534053F82F0FB66FFA41088B", {3, 2, 4, 1, 5, 0, 6}), 8: ("EA173944A06C275C8E8614D629A527B82B2F8E8828E3B05C6B7F16B1E82FE32B", {4, 2, 1, 0, 5, 3, 6}), 10: ("0F6146F5A7E3D982156552957CB794CA86EB70A36BD62490C8098CCDE13AAEF8", {6, 2, 4, 0, 5, 3, 1}), 9: ("78CB32222F004EFAA301CEDCD3429E0E8017912C047B43E22D4D738A083B361C", {5, 4, 0, 2, 3, 1, 6}), 6: ("479BB3B603DC19D1E8371F945D4C5923B20E2CB49639B57EFC257A0330BAFF4D", {5, 4, 2, 3, 1, 6, 0})}
7
reason: all committed
done!
